src/main/java/mate/academy/service/impl/MovieSessionServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.service.impl;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.dao.MovieSessionDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.MovieSession;
+import mate.academy.service.MovieSessionService;
+
+@Service
+public class MovieSessionServiceImpl implements MovieSessionService {
+    @Inject
+    private MovieSessionDao movieSessionDao;
+
+    @Override
+    public MovieSession add(MovieSession movieSession) {
+        return movieSessionDao.add(movieSession);
+    }
+
+    @Override
+    public MovieSession get(Long id) {
+        return movieSessionDao.get(id).get();

```

Comment:
Please use .orElseThrow() method with meaningful exception instead of raw usage or .get() to produce more understandable result when entity wasn't found.
Please recheck all occurences